src/main/java/fr/openent/viescolaire/core/enums/DayOfWeek.java

```diff
@@ -0,0 +1,11 @@
+package fr.openent.viescolaire.core.enums;
+
+public enum DayOfWeek {
+    MONDAY,
+    TUESDAY,
+    WEDNESDAY,
+    THURSDAY,
+    FRIDAY,
+    SATURDAY,
+    SUNDAY
+}
```

Comment:
Voir comm dans src/main/java/fr/openent/viescolaire/model/InitForm/InitFormTimetable.java
```suggestion
public enum DayOfWeek {
    MONDAY,
    TUESDAY,
    WEDNESDAY,
    THURSDAY,
    FRIDAY,
    SATURDAY,
    SUNDAY;

    public static DayOfWeek getValue(String value) {
        return getValue(value, null);
    }

    public static DayOfWeek getValue(String value, DayOfWeek defaultValue) {
        return Arrays.stream(DayOfWeek.values())
                .filter(dayOfWeek -> dayOfWeek.name().equalsIgnoreCase(value))
                .findFirst()
                .orElse(defaultValue);
    }
}
```